attribute-utils/src/main/java/org/hypertrace/attribute/db/DbAttributeUtils.java

```diff
@@ -0,0 +1,156 @@
+package org.hypertrace.attribute.db;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.hypertrace.core.datamodel.AttributeValue;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.shared.SpanAttributeUtils;
+import org.hypertrace.core.span.constants.RawSpanConstants;
+import org.hypertrace.core.span.constants.v1.Mongo;
+import org.hypertrace.core.span.constants.v1.Redis;
+import org.hypertrace.core.span.constants.v1.Sql;
+
+public class DbAttributeUtils {
+
+  // db related OTEL attributes
+  private static final String OTEL_DB_SYSTEM = OTelDbAttributes.DB_SYSTEM.getValue();
+  private static final String OTEL_DB_CONNECTION_STRING = OTelDbAttributes.DB_CONNECTION_STRING.getValue();
+  private static final String OTEL_DB_OPERATION = OTelDbAttributes.DB_OPERATION.getValue();
+  private static final String OTEL_NET_PEER_IP = OTelDbAttributes.NET_PEER_IP.getValue();
+  private static final String OTEL_NET_PEER_PORT = OTelDbAttributes.NET_PEER_PORT.getValue();
+  private static final String OTEL_NET_PEER_NAME = OTelDbAttributes.NET_PEER_NAME.getValue();
+  private static final String OTEL_NET_TRANSPORT = OTelDbAttributes.NET_TRANSPORT.getValue();
+
+  // mongo specific attributes
+  private static final String OTEL_MONGO_DB_SYSTEM_VALUE = OTelDbAttributes.MONGODB_SYSTEM_VALUE.getValue();
+  private static final String OTHER_MONGO_ADDRESS = RawSpanConstants.getValue(Mongo.MONGO_ADDRESS);
+  private static final String OTHER_MONGO_URL = RawSpanConstants.getValue(Mongo.MONGO_URL);
+  private static final String OTHER_MONGO_OPERATION = RawSpanConstants.getValue(Mongo.MONGO_OPERATION);
+  private static final String OTHER_MONGO_NAMESPACE = RawSpanConstants.getValue(Mongo.MONGO_NAMESPACE);
+  private static final String OTEL_MONGO_COLLECTION = OTelDbAttributes.MONGODB_COLLECTION.getValue();
+  //private static final AttributeValue MONGO_DEFAULT_PORT = EnricherUtil.createAttributeValue(EnricherUtil.createAttributeValue("27017"));
+
+  // redis specific attributes
+  private static final String OTHER_REDIS_CONNECTION = RawSpanConstants.getValue(Redis.REDIS_CONNECTION);
+  private static final String OTEL_REDIS_DB_SYSTEM_VALUE = "redis";
```

Comment:
Q: Can we follow the same pattern? e.g for mongo we have enum - OTelDbAttributes.MONGODB_SYSTEM_VALUE, but for Redis, we have a string constant. 

Q: Do we need an enum class - OTelDbAttributes.MONGODB_SYSTEM_VALUE?

Reply:
1. Done
2. The convention I am following is to have enum class per format (OTEL etc.)